qa/tasks/cephfs/test_exports.py

```diff
@@ -131,12 +131,12 @@ def test_export_pin_getfattr(self):
             p = self.mount_a.client_remote.sh('uname -r', wait=True)
             dir_pin = self.mount_a.getfattr("1", "ceph.dir.pin")
             log.debug("mount.getfattr('1','ceph.dir.pin'): %s " % dir_pin)
-            if str(p.stdout.getvalue()) < "5" and not(dir_pin):
+            if str(p) < "5" and not(dir_pin):
                 self.skipTest("Kernel does not support getting the extended attribute ceph.dir.pin")
-        self.assertTrue(self.mount_a.getfattr("1", "ceph.dir.pin") == "1")
-        self.assertTrue(self.mount_a.getfattr("1/2", "ceph.dir.pin") == "0")
+        self.assertEqual(self.mount_a.getfattr("1", "ceph.dir.pin"), "1")
```

Comment:
i am confused. in this change you backport a change which breaks the test, and then create a commit which fixes it.

Reply:
@tchaikov so you suggest that I alter the cherry-pick commit itself (IOW merge the commits)? and then mention in the cherry-pick commit message that I'd to make some changes to make it work with nautilus?

Reply:
@tchaikov Only the `"1" -> b'1'` change is what breaks it. The rest of the changes in that commit are necessary. Hence the subsequent commit to fix just that.

Reply:
no, i think what you need is 

1. backport b98f3e388b6a38e91c2ee064466ae8f130a5bcda as this PR already have it
1. backport 3c1330795add81126424e9e0a49cc07881fbdf8f

Reply:
Ah! Got it! :-) Thanks a lot @tchaikov